### DISCUSSION TITLE
[ 功能建议 ]archery新增http隧道连接数据库的模式

### DISCUSSION THREAD

**Discussion by wqfgithub** [hhyo/Archery#1605] [General]
### 目前遇到的问题/使用障碍
由于线上环境DB无公网IP，不支持SSH跳转。无法使用现有模式连接
### 希望如何解决/实现它
在允许开放有限端口，目前我们Navicat通过http隧道进行连接管理。希望archery可以支持该http隧道连接数据库模式
### 其他信息
如果有其他类似的产品功能或者图片信息，可在此提交

**Comment by waitspring** [comment]
archery 常规来说本就应该部署在产线环境吧，为什么archery 会走公网访问数据库呢？

**Comment by wqfgithub** [comment]
> archery 常规来说本就应该部署在产线环境吧，为什么archery 会走公网访问数据库呢？
我们在内网部署一套archery环境，生产环境几十套分别在自己对应公有云或者私有云里面，希望通过内网archery直接远程连接访问
